blueprints/in-repo-engine/files/lib/__name__/index.js

```diff
@@ -0,0 +1,8 @@
+/*jshint node:true*/
+module.exports = {
+  name: '<%= dasherizedModuleName %>',
+
+  isDevelopingAddon: function() {
+    return true;
+  }
+};
```

Comment:
We also had to disable hinting for the in-repo engines in this repo (see https://github.com/dgeb/ember-engines/blob/master/tests/dummy/lib/ember-chat/index.js#L11) to prevent errors from being thrown. Can you test if that is needed here also (I'm hoping it is not needed, but there may be bugs to fix upstream in ember-cli)?